src/XCCDF_POLICY/xccdf_policy.c

```diff
@@ -1000,9 +1000,9 @@ _xccdf_policy_rule_evaluate(struct xccdf_policy * policy, const struct xccdf_rul
 	xccdf_role_t role = xccdf_get_final_role(rule, r_rule);
 
 	if (!is_selected) {
-		dI("Rule '%s' is not selected.", rule_id);
 		return _xccdf_policy_report_rule_result(policy, result, rule, NULL, XCCDF_RESULT_NOT_SELECTED, NULL);
 	}
+	dI("Evaluating XCCDF rule '%s'.", rule_id);
```

Comment:
This is being printed for all rules and groups even when I use `--rule`. Can we do something about it?

Reply:
Let me check

Reply:
Hmm. `./run utils/oscap --verbose=DEVEL xccdf eval --profile xccdf_org.ssgproject.content_profile_ospp --rule=xccdf_org.ssgproject.content_rule_no_empty_passwords ../../content/build/ssg-fedora-ds.xml` gives me exactly one line `I: oscap: Evaluating XCCDF rule 'xccdf_org.ssgproject.content_rule_no_empty_passwords'...`.

Reply:
@matusmarhefka Can you please give me an example of a command to use to reproduce behaviour observed by you?

Reply:
@evgenyz Okay, this was probably something wrong on my side as now I have tried to re-test it and the message is not printed anymore. Before, I have used `make install` and used `oscap` instead of the `./run utils/oscap`, but now it works correctly for both.